### PR TITLE
🎭 Restyle admin form

### PR DESCRIPTION
**Pull request by kenetec**
Restyles admin form with Carousel element and adds new hours of operation sub-form.

### CODE REVIEW

**Comment by n3a9** [comment]
This looks awesome! Thank you kendall :) 

A couple things:

1. Can you make the buttons rounded like on the home page? The css is `border-radius: 4px`

2. Is it possible to hide the "Add resource" button until the very end, where the next will turn into add resouce? that way, it also doesn't cycle back to the first page

**Comment by jackieo5023** [comment]
> This looks awesome! Thank you kendall :)
> 
> A couple things:
> 
> 1. Can you make the buttons rounded like on the home page? The css is `border-radius: 4px`
> 2. Is it possible to hide the "Add resource" button until the very end, where the next will turn into add resouce? that way, it also doesn't cycle back to the first page

Also maybe a back button

File: client/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "homepage": "https://ymca.now.sh/",
   "dependencies": {
+    "@material-ui/pickers": "^3.2.10",
```
Comment (jackieo5023) [inline client/package.json]: we can remove this dependency

File: client/src/components/LabelWrapper.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  label: String,
+  required: any,
+  component: any => any,
+};
+
+const LabelWrapper = (props: LabelWrapperProps) => {
+  const { label, required, component } = props;
+
+  return (
+    <div>
```
Comment (jackieo5023) [inline client/src/components/LabelWrapper.js]: can be <> instead

File: client/src/components/ResourceHoursOfOperationForm.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
+// @flow
+import React, { useCallback } from 'react';
+import '../css/ResourcePhoneNumberForm.css';
+import { Form, Button, Row, Col, TimePicker } from 'antd';
+
+// import LabelWrapper from './LabelWrapper';
+// import { KeyboardTimePicker } from "@material-ui/pickers";
```
Comment (jackieo5023) [inline client/src/components/ResourceHoursOfOperationForm.js]: we can remove these

File: client/src/css/AdminResourceManager.css [hunk 4]
```diff
@@ -1,4 +0,0 @@
-.admin-resource-form {
```
Comment (jackieo5023) [inline client/src/css/AdminResourceManager.css]: does this file have nothing in it then?

Comment (kenetec) [reply client/src/css/AdminResourceManager.css]: ye

File: client/src/css/ResourceForm.css [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+.carousel-move-btn {
+  margin: auto;
+  background: #6bdbd0 !important;
+  color: white !important;
+  border-radius: 0px !important;
+  box-shadow: none !important;
+  border: 0px !important;
+  border-radius: 4px !important;
+  /* margin-bottom: 3em; */
+  /* margin-left: 44vw; */
```
Comment (jackieo5023) [inline client/src/css/ResourceForm.css]: We should probably remove the comments

**Pull request merged**